matRad_electronDensitiesToHU.m

```diff
@@ -0,0 +1,59 @@
+function ct = matRad_electronDensitiesToHU(ct)
+
+% load hlut
+
+% directory with look up table files

```

Comment:
The entire loading process of the HLUT appears twice, i.e. here within this very funciton and in the HUToElectronDens. We should put this into one function so we only have to maintain this code once.